src/222/main/kotlin/org/rust/lang/core/psi/ext/IntentionPreview.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import com.intellij.psi.PsiElement
+
+val PsiElement.isIntentionPreviewElement: Boolean get() = false
```

Comment:
```suggestion
val PsiElement.isIntentionPreviewElement: Boolean get() = !isPhysical
```

Reply:
I think it should be `false` not to break something. The intention preview will not work in 2022.2, but it is anyway disabled there by default. E.g. we have `RsImplTraitMemberCompletionProvider` which calls `import` on non-physical `PsiElement` (from copy created in completion). This particular case is probably a bug and should be fixed, but potentially there are other cases not covered by tests

Reply:
Ok